<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>left</class>
 <widget class="QWidget" name="left">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="openDir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Open
Dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Next
Image</string>
   </property>
   <property name="shortcut">
    <string>D, Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_Image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev
Image</string>
   </property>
   <property name="shortcut">
    <string>A, Left</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
